Realm.Shared/native/Configuration.cs

```diff
@@ -26,6 +27,7 @@ namespace Realms.Native
     internal delegate bool MigrationCallback(IntPtr oldRealm, IntPtr newRealm, Schema oldSchema, ulong schemaVersion, IntPtr managedMigrationHandle);
 
     [StructLayout(LayoutKind.Sequential)]
+    [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1307:AccessibleFieldsMustBeginWithUpperCaseLetter")]
```

Comment:
I don't think StyleCop supports that :/ It supports merging rule configs with parent folders, but only up to the project level. Once it reaches that, it doesn't merge with child settings, which prevents us from applying it in the Native folder either.